File: Arithmetic_FastHCF.pl
#!/usr/bin/perl;
use strict;
use warnings;
use FASTHCF qw(getHCF);

# Useful variables
my $answer;
my @set;

# Use the fast HCF module/class
@set = (20, 30, 40);
my $h_c_f = FASTHCF->new(\@set);
$answer = $h_c_f->getHCF();
print ("The H.C.F. of ", join(", ", @set), " is $answer\n");


print "\n\n";

File: Arithmetic_FastHCF.vb
Module Arithmetic_fast_HCF

    Sub Main()

        ' Use the fast HCF Class
        Dim group As New List(Of Integer)({20, 30, 40})
        Dim h_c_f As New FastHCF
        h_c_f._init_(group)
        Dim answer = h_c_f.getHCF()

        Console.Write("The H.C.F. of " & String.Join(", ", group) & " is " & answer)

    End Sub

End Module
